### ISSUE TITLE
Unable to enable health check for lambda target group

### ISSUE THREAD

**Issue by jeevesdavid** [terraform-providers/terraform-provider-aws#6963]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* aws_lb_target_group

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```
resource "aws_lb_target_group" "scim-tg" {
  name        = "${local.resource_name_prefix}-tg"
  target_type = "lambda"
  
  health_check{
    path = "/users/_healthcheck"
    interval = "5"
    timeout = "4"
  }

}

### Expected Behavior

We should be able to enable or disable health check for lambda target group. 

### Actual Behavior

The health check parameters are applied but health check is disabled and there is no way to enable it via terraform.

![image](https://user-images.githubusercontent.com/8177569/50359653-7e2b2480-052b-11e9-8060-cd1cab35f951.png)

### Steps to Reproduce

1. `terraform apply`

**Comment by bflad** [comment]
Can confirm this is a new boolean argument that is required for the new type of Target Group.

From the ELBv2 User Guide: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/lambda-functions.html#enable-health-checks-lambda

API reference: https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_CreateTargetGroup.html
> **HealthCheckEnabled**
> Indicates whether health checks are enabled. If the target type is instance or ip, the default is true. If the target type is lambda, the default is false.
> 
> Type: Boolean
> 
> Required: No

**Comment by sunilkumarmohanty** [comment]
Is it ok to  just enable health check if the health_check attribute is present for lambda?

**Comment by jeevesdavid** [comment]
Healthcheck can be enabled or disabled from the console and it is disabled by default. So I think it should be disabled by default and enabled if the health_check attribute is enabled. 

In addition it will be useful to have an explicit option to disable the healthcheck even if the health_check attribute is specified.

**Comment by marcohutzsch1234** [comment]
Any news on this issue?

**Comment by karlismelderis** [comment]
👍 I just got bitten by this issue

**Comment by nywilken** [comment]
The new `eanbled` argument for the `aws_lb_target_group` resource has been merged and will release with version 2.5.0 of the Terraform AWS Provider later today.

**Comment by bflad** [comment]
This has been released in [version 2.5.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#250-april-05-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
